### DISCUSSION TITLE
Unable to make Outlook connection work with Security Default Setting On

### DISCUSSION THREAD

**Discussion by shubhambharti89** [emailjs-com/help#44] [Q&A]
I am using EmailJS to get form details sent to my email. When using Outlook Connector with Default Security Setting disable in Outlook - everything works fine. 

But now management asked that we need to enable Default security setting. After enabling that - and creating app password, test email is throwing below error, I tried unchecking test email - get the same error on prod also :

**Outlook: Invalid login: 535 5.7.139 Authentication unsuccessful, the user credentials were incorrect.**

Authenticated SMTP is enabled on the Mail.

If I try to use Outlook V2 connector and connecting it to the email account and successfully sending test email, it throws a different error for production queries:

**Outlook: A participant without an email address is not allowed for ReplyTo property.**

Can anyone help on what I am doing wrong?

**Comment by xr0master** [comment]
Hi,
> But now management asked that we need to enable Default security setting.

This is the correct decision and should not affect performance.

> Outlook: Invalid login: 535 5.7.139 Authentication unsuccessful, the user credentials were incorrect.

Please make sure you enter the correct App Password. Although the first version will soon be deprecated.

> Outlook: A participant without an email address is not allowed for ReplyTo property.

And what do you have in the "Reply To" field?
